resources/testharness.js

```diff
@@ -3667,7 +3673,10 @@ policies and contribution forms [3].
     if (global_scope.addEventListener) {
         var error_handler = function(message, stack) {
             if (tests.tests.length === 0 && !tests.allow_uncaught_exception) {
-                tests.set_file_is_test();
+                tests.status.status = tests.status.ERROR;
+                tests.status.message = "An error occured before testing began.";
```

Comment:
Let’s just delete this code and let the rest of the code do it’s thing, that would still result in an harness error and have a better error message.

Reply:
You got it